aiida/cmdline/commands/cmd_import.py

```diff
@@ -35,8 +48,18 @@
     type=GroupParamType(create_if_not_exist=True),
     help='Specify group to which all the import nodes will be added. If such a group does not exist, it will be'
     ' created automatically.')
+@click.option(
+    '-e',
+    '--extras_import_mode',
+    type=click.Choice(EXTRAS_IMPORT_MODES),
+    default='keep_existing',
+    help="keep_existing: in case of a name collision keep the original value. "
```

Comment:
Suggestion to clarify the help string of the option:
```
"Specify which extras from the export archive should imported for nodes that are already contained in the database: "
"ask: import all extras and prompt what to do for existing extras. "
"keep_existing: import all extras and keep original value of existing extras. "
"update_existing: import all extras and overwrite value of existing extras. "
"mirror: import all extras and remove any existing extras that are not present in the archive. "
"none: do not import any extras. "
```

Reply:
done